### PR TITLE
Fix request queue size metric and add response queue metrics

### PR DESCRIPTION
**Pull request by hangc0276**
### Motivation
Some request queue size metric doesn't use correct metric type.

### Modification
1. Fix request queue size metric type
2. Add response queue metric to leverage whether keep order operation cost too much time.

### CODE REVIEW

File: kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/RequestStats.java [hunk 1]
```diff
@@ -45,11 +50,9 @@
...
 @Getter
 public class RequestStats {
-    @StatsDoc(
-            name = REQUEST_QUEUE_SIZE,
-            help = "request queue's size"
-    )
-    private final Counter requestQueueSize;
+
+    final AtomicInteger requestQueueSize = new AtomicInteger(0);
+    final AtomicInteger responseQueueSize = new AtomicInteger(0);
```
Comment (BewareMyPower) [inline kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/RequestStats.java:55]: Use `private`

**Review by BewareMyPower — CHANGES_REQUESTED** [review]

**Review by jiazhai — APPROVED** [review]
overall lgtm 👍

**Comment by BewareMyPower** [comment]
LGTM after the Codacy check was resolved.

Some codes are conflict with my PR https://github.com/streamnative/kop/pull/500, I'll rebase and fix the conflict if this PR was merged first.

File: kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/RequestStats.java (hunk 1 above)
Comment (BewareMyPower) [inline kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/RequestStats.java:55]:
```suggestion
    private final AtomicInteger requestQueueSize = new AtomicInteger(0);
    private final AtomicInteger responseQueueSize = new AtomicInteger(0);
```

**Review by BewareMyPower — CHANGES_REQUESTED** [review]

**Review by BewareMyPower — APPROVED** [review]

**Pull request merged**